events/ready.js

```diff
@@ -29,7 +29,7 @@ module.exports = class {
     setInterval(() => {
       const toRemind = this.client.reminders.filter(r => r.reminderTimestamp <= Date.now());
       toRemind.forEach(reminder => {
-        this.client.users.get(reminder.id).send(`You asked me to remind you about: \`${reminder.reminder}\``);
+        this.client.users.get(reminder.id).send(`You asked me to remind you about: \`${reminder.reminder}\` in \`${this.client.guilds.get(reminder.guildid).name}\``);
```

Comment:
Probably a check in case the bot left the guild, for safety.

Reply:
I already delete all reminders for any guild Misaki is removed from [here](https://github.com/YorkAARGH/Misaki/blob/master/events/guildDelete.js#L17-L19)

Reply:
In that case, I'll approve this.